content/guides/getstarted/how-to-create-a-token.md

```diff
@@ -166,7 +166,7 @@ We will use our `mnt...` address as the
 [token mint account](/docs/core/tokens.md#mint-account) for our new token:
 
 ```bash
-spl-token create-token --program-id TokenzQdBNbLqP5VEhdkAS6EPFLC1PHnBqCXEpPxuEb --enable-metadata mntsBJFZE48XnQteT1vekH35UXDfamtdpFv6Xh8A7Gy.json
+spl-token create-token --program-id TokenzQdBNbLqP5VEhdkAS6EPFLC1PHnBqCXEpPxuEb --enable-metadata mntDh6s9hd2wbbS6ZWaYTCC9GXxhXGvbWpUgLt5qYgv.json
```

Comment:
```suggestion
spl-token create-token --program-id TokenzQdBNbLqP5VEhdkAS6EPFLC1PHnBqCXEpPxuEb --enable-metadata mntTymSqMU4e1NEDdxJ9XoPN4MitCgQ7xxGW6AuRAWQ.json
```

Reply:
Did you change all of them to the correct hey? 
mntDh6s9hd2wbbS6ZWaYTCC9GXxhXGvbWpUgLt5qYgv does not exist on devnet
all of the keys should be mntTymSqMU4e1NEDdxJ9XoPN4MitCgQ7xxGW6AuRAWQ 

I was talking about line 141